package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.21.0",
+  "version": "0.22.0",
```

Comment:
Please revert the version change - we'd like to change it ourselves when we create the next release (which will be really soon, promised 👍 )